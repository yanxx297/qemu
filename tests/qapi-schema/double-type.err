tests/qapi-schema/double-type.json:2: Unknown key 'command' in type 'bar'
