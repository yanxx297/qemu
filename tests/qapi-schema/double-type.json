# FIXME: we should reject an expression with ambiguous metatype
{ 'command': 'foo', 'type': 'bar', 'data': { } }
