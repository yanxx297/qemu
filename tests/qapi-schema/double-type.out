[OrderedDict([('command', 'foo'), ('type', 'bar'), ('data', OrderedDict())])]
[]
[OrderedDict([('command', 'foo'), ('type', 'bar'), ('data', OrderedDict())])]
